scrapers/ep_votes/scrapers.py

```diff
@@ -611,7 +611,13 @@ def _session(self, session: dict) -> Session:
         )
 
     def _parse_date(self, day: str) -> date:
-        return date.fromisoformat(str(self.year) + "-" + str(self.month) + "-" + day)
+        return date.fromisoformat(
+            str(self.year).rjust(2, "0")
+            + "-"
+            + str(self.month).rjust(2, "0")
+            + "-"
+            + day
```

Comment:
Should we maybe add leading zeros here as well, just in case that the source we crawl changes its format?

```suggestion
            + day.rjust(2, "0")
```